domain/modeldefaults/state/state.go

```diff
@@ -121,11 +121,54 @@ WHERE model_metadata.model_uuid = ?
 	})
 
 	if err != nil {
-		return nil, err
+		return nil, domain.CoerceError(err)
 	}
 	return rval, nil
 }
 
+// ModelMetadataDefaults is responsible for returning model config values
+// related to the models metadata. If no model is found for the provided uuid an
```

Comment:
you know what I'm going to say here.....

Reply:
Done & Done